components/dashboard/src/menu/OrganizationSelector.tsx

```diff
@@ -82,8 +82,15 @@ export default function OrganizationSelector() {
         });
     }
 
-    // Show settings if user is an owner of current org
+    // Show billing & settings if user is an owner of current org
     if (currentOrg.data && currentOrg.data.isOwner) {
+        linkEntries.push({
+            title: "Billing",
+            customContent: <LinkEntry>Billing</LinkEntry>,
+            active: false,
+            separator: false,
+            link: "/billing",
+        });
```

Comment:
**thought**: This will not follow the hierarchy of the current placement of billing but will make billing more visible, see linked discussion in the PR description.